OpenRA.Mods.Common/Widgets/Logic/SettingsLogic.cs

```diff
@@ -50,8 +53,13 @@ enum PanelType { Display, Audio, Input, Hotkeys, Advanced }
 		static SettingsLogic()
 		{
 			var original = Game.Settings;
+
+			// hardware changes are able to invalidate the stored value so get the current display
```

Comment:
Style nit: Please start comments with capital letters (also applies to other comments in the PR).